ironfish/src/rpc/routes/wallet/getTransaction.ts

```diff
@@ -98,10 +100,14 @@ router.register<typeof GetAccountTransactionRequestSchema, GetAccountTransaction
         ? decryptedNoteForOwner.note
         : new Note(decryptedNote.serializedNote)
 
+      const asset = await node.chain.getAssetById(note.assetId())
+
       serializedNotes.push({
         owner,
         memo: note.memo(),
         value: CurrencyUtils.encode(note.value()),
+        assetId: note.assetId().toString('hex'),
+        assetName: asset?.name.toString('utf8') || '',
```

Comment:
Buffers should remain buffers through our RPC layer, and rendered on the CLI side.

Reply:
Discussed elsewhere; we'll merge these changes as they are for now and change strings to buffers in RPC in a separate PR.